File: Session_01_BasicPagramming/Lab01/Python_Lang/Demo08.py
#
#  Created by NDHuy at 09/04/2020
#  Copyright 2020 by SEstudio
#  Taxi payment
#

def main():
    n = int(input("Input number of taxi km n = "))
    total_price = 0.0

    if n > 0:
        if n == 1:
            total_price = 15000.0
        elif n <= 5:
            total_price = 15000.0 + ((n - 1) * 13500.0)
        else:
            total_price = 15000.0 + (4 * 13500.0) + ((n - 5) * 11000.0)
            if n > 120:
                total_price = total_price * 0.9
    
    print("Total price: %.0f" %(total_price))
    

if __name__ == "__main__":
    main()

File: Session_01_BasicPagramming/Lab01/C_Lang/src/Demo08.cpp
//
//  Created by NDHuy at 09/04/2020
//  Copyright 2020 by SEstudio
//  Taxi payment
//

#include <stdio.h>

int main()
{
    int n; 
    printf("Input number of taxi km n = ");
    scanf("%d", &n);

    float total_price = 0.0;
    
    if(n > 0)
    {
        total_price = 15000.0;
        n = n - 1;

        if(n > 0)
        {
            int t = (n >= 4) ? 4 : n;
            
            total_price += t * 13500.0;

            if(n > t)
                n = n - t;

            total_price += n * 11000.0;

            // Discount
            if (n > 115)
                total_price = total_price * 0.9;
        }
    }

    printf("Total price: %.0f \n", total_price);

    return 0;
}